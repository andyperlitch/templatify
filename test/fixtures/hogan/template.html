Hello {{name}}
You have just won ${{value}}!
{{#in_ca}}
Well, ${{taxed_value}}, after taxes.
{{/in_ca}}

partial:
{{> app_template_partial }}

Items:
{{#items}}
  Here's an item: {{name}} - {{value}}

  Parts:
  {{#parts}}
    {{name}} - {{value}}

    Item partial:
    {{> app_template_item }}
  {{/parts}}
{{/items}}

{{hash.key}}
