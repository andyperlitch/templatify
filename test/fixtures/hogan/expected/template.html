Hello Chris
You have just won $10000!
Well, $6000, after taxes.

partial:
<p>I am a partial</p>

with another one:

<p>I am another partial</p>

<div class="with nested partial">
  with another partial again - Chris
  <p>YAY!</p>
</div>

Items:
  Here's an item: foobar - 10000

  Parts:
    foo - 10000

    Item partial:
    <p>Hi, I'm a partial within an list of item</p>

name: foo
val: bar
value: 10000

